src/components/Offramp/Confirm.view.tsx

```
@@ -646,6 +654,53 @@ export const OfframpConfirmView = ({
+    // check if fee exceeds withdraw amount and update error state
+    useEffect(() => {
+        const totalFees = calculateTotalFees(estimatedGasCost, !!appliedPromoCode, accountType, !!crossChainDetails)
+
+        const amount =
+            offrampType == OfframpType.CASHOUT
+                ? parseFloat(usdValue ?? '0')
+                : tokenPrice && claimLinkData
+                  ? tokenPrice * parseFloat(claimLinkData.tokenAmount)
+                  : 0
+
+        if (!isNaN(amount) && !isNaN(totalFees) && amount <= totalFees) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)